### PR TITLE
add eid uptime monitor

### PR DESCRIPTION
**Pull request by phuesler**

### CODE REVIEW

File: .upptimerc.yml [hunk 1]
```diff
@@ -25,6 +25,11 @@ sites:
     icon: https://www.steuerlotse-rente.de/favicon.ico
     maxResponseTime: 5000
 
+  - name: eid
+    url: https://eid.digitalservicebund.de/actuator/health
+    icon: https://digitalservice.bund.de/favicon-16x16.png
```
Comment (pgurusinga) [inline .upptimerc.yml:30]: I already have a favicon from designer, I will upload and let you know.

File: .upptimerc.yml [hunk 2]
```diff
@@ -25,6 +25,11 @@ sites:
     icon: https://www.steuerlotse-rente.de/favicon.ico
     maxResponseTime: 5000
 
+  - name: eid
+    url: https://eid.digitalservicebund.de/actuator/health
+    icon: https://eid.digitalservicebund.de/images/favicon.png
+    maxResponseTime: 5000
+
```
Comment (pgurusinga) [inline .upptimerc.yml:32]: do we need an assignee? 
https://github.com/digitalservicebund/uptime-monitor/blob/3118b1017f49a5505e274b3b089dcd2971faeea1/.upptimerc.yml#L20-L21

**Review by pgurusinga — APPROVED** [review]

File: .upptimerc.yml (hunk 2 above)
Comment (phuesler) [reply .upptimerc.yml:32]: Not yet. Once things are wired up in the background

**Comment by nfelger** [comment]
LGTM - Just be aware that this will only send alerts to Slack and there's no straightforward way to feed alerts into OpsGenie.

**Pull request merged**